AddSevenPS/Library.fs

```diff
@@ -4,16 +4,17 @@ open System.Management.Automation
 
 open System.Management.Automation.Runspaces
 
-// Cmdlet attribute declaration
-[<Cmdlet("Add", "Seven")>] 
+module AddSeven =
+    let addSeven num = num + 7
 
```

Comment:
It seems that using a `module` structure, we can import this function within the `type` as well as within the test.